pages/get-involved.tsx

```diff
@@ -3,20 +3,115 @@ import {
 import FactCard from '../components/layout/FactCard.tsx';
 import HeroBanner from '../components/layout/HeroBanner.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import PaperSection from '../components/layout/PaperSection.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/get-involved.ts';
 
+function BenefitsOfJoiningUs() {
+  const theme = useTheme();
+  const matchesXS = useMediaQuery(theme.breakpoints.down('sm'));
+  const matchesSM = useMediaQuery(theme.breakpoints.down('md'));
+
+  let bottomImageAdjust;
+  if (matchesSM) {
+    bottomImageAdjust = '9.8rem';
+  } else {
+    bottomImageAdjust = '6.6rem';
+  }
+  return (
+    <SectionContainer
+      id="benefits-of-joining-us"
+      maxWidth="md"
+    >
+      <Typography variant="h2">
+        {content.benefits.header}
+      </Typography>
+      <PaperSection sx={{
+        bgcolor: 'primary.dark', color: 'primary.contrastText',
+      }}
+      >
+        <Box sx={{
+          pl: 3, py: 4,
+        }}
+        >
+          {content.benefits.body.map((text) => (
+            <Typography
+              key={`benefitsText-${text}`}
+              variant="body1"
+              sx={{ mb: 4, mr: 2 }}
+            >
+              {text}
+            </Typography>
+          ))}
+          {matchesXS ? (
+            <Grid container alignItems="flex-start">
+              <Grid item xs={5}>
+                <Typography variant="body1" paragraph>
+                  {content.benefits.listHeader}
+                </Typography>
+              </Grid>
+              <Grid item xs={7}>
+                <ImageContainer
+                  src={content.benefits.mobileImgsrc}
+                  width={95}
+                  height={95}
+                  alt=""
+                  useImageDimensions
+                />
+              </Grid>
+            </Grid>
+          )
+            : (
+              <Typography variant="body1" paragraph>
+                {content.benefits.listHeader}
+              </Typography>
+            )}
+          <Box sx={{
+            position: 'relative', width: '100%', height: 'auto',
+          }}
+          >
+            <List>
+              {content.benefits.benefitItems.map((benefit) => (
+                <BenefitListItem key={`benefitItem-${benefit.id}`} isSmallScreen={matchesXS} {...benefit} />
+              ))}
+            </List>
+            {!matchesXS
+            && (
+              <ImageContainer
+                src={content.benefits.imgsrc}
+                alt=""
+                width={200}
+                height={200}
+                style={{
+                  position: 'absolute',
+                  right: '3rem',
+                  bottom: `${bottomImageAdjust}`,
+                  transition: 'right 0.5s top 0.5s',
+                }}
+                useImageDimensions
+              />
+            )}
```

Comment:
```suggestion
            <Box
              sx={{
                  position: 'absolute',
                  right: '3rem',
                  bottom: { sm: `6.6rem`, md: `9.8rem` },
                  transition: 'right 0.5s top 0.5s',
                }}
            >
              <ImageContainer
                src={content.benefits.imgsrc}
                alt="Alt text here"
                width={200}
                height={200}
                useImageDimensions
              />
           </Box>
```
Here again we can just use breakpoint that `sx` give to us insted of create some kind of new logic to manage this.